pyracing/response_objects/historical_data.py

```diff
@@ -3,89 +3,89 @@
 
 class EventResults:
     def __init__(self, dict):
-        self.bestlaptime = parse_encode(dict['38'])
-        self.bestquallaptime = parse_encode(dict['6'])
-        self.carclassid = dict['29']
-        self.carid = dict['32']
-        self.catid = dict['33']
-        self.champpoints = dict['40']
-        self.champpointssort = dict['43']
-        self.clubpoints = dict['16']
-        self.clubpointssort = dict['47']
-        self.custid = dict['23']
-        self.displayname = parse_encode(dict['48'])
-        self.dropracepoints = dict['17']
-        self.evttype = dict['37']
-        self.finishedat = dict['10']
-        self.finishing_position = dict['3']
-        self.groupname = dict['19']
+        self.best_qlap = parse_encode(dict['38'])
```

Comment:
not sure I like this one. is it best qualifing lap? Maybe it should be `best_qual_lap` or something

Reply:
lap times are often referred to as blap, qlap, and olap. best_lap, qual_lap, and optimal_lap, respectively. Telemetry services offer an 'olap' for each series for you to download and compare your laps against. I think this is a pretty safe one to abbreviate